### DISCUSSION TITLE
Sharing album

### DISCUSSION THREAD

**Discussion by smolesen** [photoprism/photoprism#3079] [Q&A] [answered]
I'm new to PhotoPrism, so maybe it just me not understanding how it's suppose to work.

I have two albums 'Album1' and 'Album2', I select Album1 and share it, the link is something like:

http://photoprism.mydomain.com/s/1gv6pzrzir/Album1

then I unselect Album1 and select Album2 and share that one one two, the link is something like:

http://photoprism.mydomain.com/s/27jfb55d4m/Album2

Now I open the first link in a browser, and I can see the content of Album1, if I click the little 'album' icon I just see Album1
Now in another browser I open the second link, and I see the content of Album2, but now if I click the little 'album' icon, I can see both Album1 and Album2, plus browse all the pictures....

I would have expected to be able to share Album1 with one group of people and Album2 with another... but is seems like they can see each others albums..... is that how it's suppose to work??

TIA

/Søren

**Comment by lastzero** [comment]
Are you using a different browser, such as Firefox instead of Chrome? If not, the additional share will be added to the list of shares so that you can see both and switch between albums. Another way to share two albums at the same time is to use the same secret.

**Comment by smolesen** [comment, marked as answer]
I'm using Chrome in Incognito mode.... 
I'd like to share the Albums to two different groups of people, and I don't want them to see each others Albums....
Is that possible?

/Søren

**Reply by lastzero** [reply]
Incognito mode sessions are shared, for example, when you use two tabs. It is basically the same browser.

**Reply by smolesen** [reply]
Ok, I'll try in two different browsers.....

/Søren

**Reply by smolesen** [reply]
Ok, seems to work as expected with two different browsers (Chrome / Edge)

Thx
/Søren
